### ISSUE TITLE
Align Pin Icon/Text to the Right side of the Node

### ISSUE THREAD

**Issue by urosidoki** [thedmd/imgui-node-editor#122]
Hi,

I am having some trouble making this work, and reading the examples doesn't seem to clarify anything for me, could someone lend me a hand on this?

Cheers!

**Comment by urosidoki** [comment]
Ok, I think I saw that you are using a modified version of ImGui, where you include all those BeginHorizontal, etc..., is that how you made it work?

**Comment by thedmd** [comment]
Yes. At first I tried to calculate necessary offsets to align widgets to the right side of the node. That became complicated quickly overwhelming any example I could make.

To solve this I made [Stack Layout](https://github.com/ocornut/imgui/pull/846) that took care of widget alignment automatically. This is single commit and I'm keeping it up to date. Now I can begin vertical layout and align every widget to the right (every widget should be separated by a Spring).

Without this extension there is no easy way to do this. At least there wasn't, now there are tables in ImGui but I didn't tested them. If they're not using child windows, should work fine. Maybe they have alignment control too.
